client/app/templates/components/contact-form.hbs

```diff
@@ -0,0 +1,36 @@
+{{#unless districtContactSubmitted}}
+  {{#unless districtContactErrored}}
+    <form {{action 'submitDistrictContact' on='submit'}}>
+        <div class="control-group">
+          <div class="control-label">
+            <label>Name</label>
+          </div>
+          <div class="controls">
+            {{input value=name name='Name' type="text"}}
+          </div>
+        </div>
+        <div class="control-group">
+            <div class="control-label">
+              <label>Email</label>
+            </div>
+            <div class="controls">
+              {{input value=districtContactEmail name='Email' required=true type="email"}}
+            </div>
+        </div>
+        <div class="control-group">
+          <div class="control-label">
+            <label>Message</label>
+          </div>
+          <div class="controls">
+            {{textarea value=message type="text" name='Message' rows="6" placeholder="We want schoolbot!"}}
+          </div>
+        </div>
+        <input type="text" name="_gotcha" style="display:none" />
+        <button type="submit"> Send </button>
+    </form>
+  {{else}}
+    <p>There was a problem submitting your email. Check your network connection and try again, or <a class='about__content__link' href="mailto:schoolbot-contact@vermonster.com?subject=SchoolBot information request">email us directly</a>.</p>
```

Comment:
I'm not sure if the `schoolbot-contact@` email address is still alive; probably better to use `schoolbot@` in any case since that's the email we've been publicizing elsewhere.